src/components/pages/WelcomeBoard/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
Looks good, however, what is the thinking behind using a class component? The React team recommends using hooks on new projects.

Reply:
Very true! Thanks for catching that